<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Плейлисты</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>20000</width>
              <height>160000</height>
             </size>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>1</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>Плейлисты</string>
             </property>
             <item>
              <property name="text">
               <string>Основной</string>
              </property>
             </item>
            </item>
            <item>
             <property name="text">
              <string>Онлайн Радио</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Локальная ФС</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QTreeView" name="treeView">
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragOnly</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="coverImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0" colspan="7">
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Плейлист</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>6</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>250</height>
             </size>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="autoScrollMargin">
             <number>6</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::DragDrop</enum>
            </property>
            <property name="defaultDropAction">
             <enum>Qt::MoveAction</enum>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideMiddle</enum>
            </property>
            <property name="gridStyle">
             <enum>Qt::NoPen</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>20</number>
            </attribute>
            <column>
             <property name="text">
              <string>Номер</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/newPrefix/num.png</normaloff>:/newPrefix/num.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Название</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/newPrefix/title.png</normaloff>:/newPrefix/title.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Артист</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/newPrefix/artist.png</normaloff>:/newPrefix/artist.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Альбом</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/newPrefix/cd_case.png</normaloff>:/newPrefix/cd_case.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Длина</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/newPrefix/length.png</normaloff>:/newPrefix/length.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Дата</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/newPrefix/date.png</normaloff>:/newPrefix/date.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Жанр</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/newPrefix/genre.png</normaloff>:/newPrefix/genre.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Размер</string>
             </property>
             <property name="icon">
              <iconset resource="../icons/icons.qrc">
               <normaloff>:/newPrefix/file_size.png</normaloff>:/newPrefix/file_size.png</iconset>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Путь</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="clearPlsButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../icons/icons.qrc">
                <normaloff>:/newPrefix/del_pls.png</normaloff>:/newPrefix/del_pls.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newPlsButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../icons/icons.qrc">
                <normaloff>:/newPrefix/add_pls.png</normaloff>:/newPrefix/add_pls.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Инфа</string>
         </attribute>
         <widget class="QWebView" name="webView">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>421</width>
            <height>231</height>
           </rect>
          </property>
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Текст</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QListWidget" name="artistList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QListWidget" name="albumList">
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>200</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="4" alignment="Qt::AlignRight">
       <widget class="QListWidget" name="titleList">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="5">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="prevButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../icons/icons.qrc">
            <normaloff>:/newPrefix/control_start_blue.png</normaloff>:/newPrefix/control_start_blue.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../icons/icons.qrc">
            <normaloff>:/newPrefix/control_play_blue.png</normaloff>:/newPrefix/control_play_blue.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../icons/icons.qrc">
            <normaloff>:/newPrefix/control_stop_blue.png</normaloff>:/newPrefix/control_stop_blue.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../icons/icons.qrc">
            <normaloff>:/newPrefix/control_end_blue.png</normaloff>:/newPrefix/control_end_blue.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="settingsButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../icons/icons.qrc">
            <normaloff>:/newPrefix/settings.png</normaloff>:/newPrefix/settings.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Phonon::VolumeSlider" name="volumeSlider"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="Phonon::SeekSlider" name="seekSlider">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="iconVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="numDigits">
           <number>8</number>
          </property>
          <property name="mode">
           <enum>QLCDNumber::Dec</enum>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
          <property name="value" stdset="0">
           <double>0.000000000000000</double>
          </property>
          <property name="intValue" stdset="0">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>885</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Библиотека</string>
    </property>
    <addaction name="actionScan"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Скрыть панель</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Сканировать библиотеку</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>Сканировать</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
